packages/run-protocol/src/vaultFactory/vaultKit.js

```diff
@@ -31,6 +31,11 @@ const wrapVault = innerVault => {
     return inner;
   };
 
+  /**
+   * Public API of the vault.
+   *
+   * @see {InnerVault} for the internal API it wraps.
```

Comment:
That's perhaps expedient, but for actual clients of this contract, referring them to the inner guts doesn't seem like the way to go.

_mostly thinking out loud..._ I suppose it's more relevant for agoric.com/documentation , but I have hopes of generating that from this.

Reply:
I agree and see that as out of scope. We need a bigger discussion around naming of contract facets.

Reply:
I'm not talking about the name of the facets. I'm talking about information hiding, or lack thereof. Clients of this API should not need to be aware of the inner API.

Reply:
Ohhh. Just this jsdoc `@see`. 

I'm not sure. Developers of agoric-sdk should know about it. End users of a GUI or REPL shouldn't (and won't see the jsdoc). What other case is there? Developers building other contracts that compose the vaultFactory contract? In that case I think it behooves them to understand the inner/outer relationship (e.g. to understand how transfers work).